.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: false
+services:
+- docker
+language: python
+cache: pip
+services:
+- docker
+
+before_install:
+- pyenv versions
+
+install:
+- pip install tox
+
+matrix:
+  include:
+  - env: TOXENV=36,coverage CODECOV=py
```

Comment:
Should we maybe add some 3.5 coverage? Quite a few folks still stuck on 3.5

Reply:
I have no problem adding 3.5, i am pretty sure the hub already works with it, let me check.

Reply:
added